keylime/migrations/versions/eb869a77abd1_create_allowlist_table.py

```diff
@@ -0,0 +1,50 @@
+"""create_allowlist_table
+
+Revision ID: eb869a77abd1
+Revises: 8da20383f6e1
+Create Date: 2021-01-12 10:54:45.263268
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'eb869a77abd1'
+down_revision = '8da20383f6e1'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+def upgrade_registrar():
+    pass
+
+
+def downgrade_registrar():
+    pass
+
+
+def upgrade_cloud_verifier():
+    op.create_table('allowlists',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('name', sa.String(255), nullable=False),
+                    sa.Column('tpm_policy', sa.Text(), nullable=True),
+                    sa.Column('vtpm_policy', sa.Text(), nullable=True),
+                    sa.Column('allowlist', sa.Text(length=429400000), nullable=True),
+                    sa.PrimaryKeyConstraint('id'),
+                    sa.UniqueConstraint('name', name='uniq_allowlists0name'),
+                    mysql_ENGINE='InnoDB',
```

Comment:
Should we have these MySQL specific options in there? The other migrations we are using don't have them. I know InnoDB is needed for things like referential integrity, etc. What does Alembic do if they are left off?

Reply:
This argument will be consumed by dialect, should have no effect when working with backends other than mysql.
The behavior depends on the default of database backend if we don't provide it, so it's a safeguard to ensure things are expected.
But the arg name seems changed, will update.